### DISCUSSION TITLE
What about Android Auto?

### DISCUSSION THREAD

**Discussion by GitHub0901** [KRTirtho/spotube#628] [Ideas]
Hi! I think that a great application like Spotube must run in Android Auto :)
I think that only needs a little modification in the manifest file, but I'm a beginer with Android :-(
I think like this link: https://developer.android.com/training/cars/media/auto?hl=es-419
See you and the work at Spotube is great!!!

**Comment by KRTirtho** [comment]
Hey that was a great resource URL and actually the Android Auto basic media playback (media session) works :rocket: :heart: 

But looks like rendering UI like Spotify isn't possible due to limited rendering capabilities of Android Auto & unlike Carplay it doesn't support 3rd party rendering like Flutter or react-native
So for the UI I've to completely rewrite  the whole app in Java/XML but I've no experience in that. Thus Spotube is only limited to basic play/pause, next/prev music. You've to manage the Queue from your mobile app :cry:

**Comment by KRTirtho** [comment]
Please follow this thread for updates https://github.com/KRTirtho/spotube/issues/623

**Comment by GitHub0901** [comment]
Great @KRTirtho 
Spotube is a great application :-) I love it!! Thanks for all the work!!!
